### PR TITLE
Setup components

### PR DESCRIPTION
**Pull request by Lordkaito**
In this branch:
1. Set up the project components needed
2. Add basic information to display in the browser
3. Deploy site on netlify

### CODE REVIEW

**Comment by helman101** [comment]
# STATUS: INVALID 🔴 ⚠️ 

Hi @Lordkaito 😢, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have linter checks, please setup the linters and try again 🚀 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

crab

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (GzimAsani) [inline .eslintrc.json:1]: - [x] As a very first note, I want to explain your confusion. At this stage of application the redux part of the application should be empty and in later stages will be used to manage the state. The redux side of the application will be used only to manage the state and not to return JSX or dom elements. Kindly check my other inline comments where I will try to explain what you should change.

File: src/components/Main.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (GzimAsani) [inline src/components/Main.js:1]: - [x] Inside the components folder there should be a a `BOOKS`(_you can name whatever you want_) component which would render all the books using the `BOOK`(_again you can name this as you like_) component
- [x] The `BOOK` component  takes the book data as a prop and renders the title and the author and has a button for removing books(currently done by you but it should be moved in the components folder side)
- [x] The AddBook component should also be moved in the component folder side and you should have two inputs one for the author and the other for the title

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Lordkaito                           
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the gitflow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [x] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**